gsoc/2019/mentors.md

```diff
@@ -90,3 +91,6 @@ layout: plain
 * Axel Naumann [axel.naumann@cern.ch](mailto:axel.naumann@cern.ch) CERN
 * Frank Wuerthwein [fkw@ucsd.edu](mailto:fkw@ucsd.edu) UCSD
 * Omar Zapata [Omar.Zapata@cern.ch](mailto:Omar.Zapata@cern.ch) OProject
+* Edward Moyse [edward.moyse@cern.ch](mailto:edward.moyse@cern.ch) University of Massachusetts, Amherst)
+* Riccardo Maria Bianchi  [riccardo.maria.bianchi@cern.ch](mailto:riccardo.maria.bianchi@cern.ch) University of Pittsburgh)
+* Andreas Salzburger  [andreas.salzburger@cern.ch](mailto:andreas.salzburger@cern.ch) CERN)
```

Comment:
I think Andreas is already present on this list, you can remove it.

Reply:
Done. BTW he is repeated earlier, as is James Catmore.

Reply:
FYI:

imacumass1:hsf.github.io emoyse$ sort gsoc/2019/mentors.md  | uniq -d

* Alaettin Serhan Mete [amete@cern.ch](mailto:amete@cern.ch) UCIrvine
* Andreas Salzburger [andreas.salzburger@cern.ch](mailto:andreas.salzburger@cern.ch) CERN
* Davide Costanzo [davide.costanzo@gmail.com](mailto:davide.costanzo@gmail.com) The University of Sheffield
* James Catmore [james.catmore@cern.ch](mailto:james.catmore@cern.ch) University of Oslo
* Paul Gessinger [paul.gessinger@cern.ch](mailto:paul.gessinger@cern.ch) CERN